### PR TITLE
Pelias autocompletion

### PR DESCRIPTION
**Pull request by hennihaus**

### CODE REVIEW

File: src/main/java/de/blackforestsolutions/dravelopsdatamodel/util/ApiToken.java [hunk 1]
```diff
@@ -99,6 +105,8 @@ public boolean getHasReferences() {
         private String path;
         private Integer maxResults;
         private String apiVersion;
+        private Box box;
+        private List<String> layers;
 
         @SuppressWarnings("CPD-START")
```
Comment (mburkertt) [inline src/main/java/de/blackforestsolutions/dravelopsdatamodel/util/ApiToken.java:111]: Warn surpression ?

File: src/main/java/de/blackforestsolutions/dravelopsdatamodel/util/ApiToken.java [hunk 2]
```diff
@@ -120,6 +128,8 @@ public ApiTokenBuilder(ApiToken apiToken) {
             this.path = apiToken.getPath();
             this.maxResults = apiToken.getMaxResults();
             this.apiVersion = apiToken.getApiVersion();
+            this.box = apiToken.getBox();
+            this.layers = apiToken.getLayers();
         }
 
         @SuppressWarnings("CPD-START")
```
Comment (mburkertt) [inline src/main/java/de/blackforestsolutions/dravelopsdatamodel/util/ApiToken.java:135]: Warn surpression ?

**Review by mburkertt — CHANGES_REQUESTED** [review]
Ich habe zwei Anmerkungen gemacht.

File: src/main/java/de/blackforestsolutions/dravelopsdatamodel/util/ApiToken.java (hunk 2 above)
Comment (hennihaus) [reply src/main/java/de/blackforestsolutions/dravelopsdatamodel/util/ApiToken.java:148]: I have made comments in code.

File: src/main/java/de/blackforestsolutions/dravelopsdatamodel/util/ApiToken.java (hunk 1 above)
Comment (hennihaus) [reply src/main/java/de/blackforestsolutions/dravelopsdatamodel/util/ApiToken.java:118]: I have made comments in code.

**Review by jobdk — APPROVED** [review]
Looks good to me. But @mburkertt should check your changes as well.

**Review by mburkertt — APPROVED** [review]
OK alles geklärt.

**Pull request merged**